mlflow/models/python_api.py

```diff
@@ -96,7 +96,7 @@ def predict(
     input_path: Optional[str] = None,
     content_type: str = _CONTENT_TYPE_JSON,
     output_path: Optional[str] = None,
-    env_manager: _EnvManager = _EnvManager.VIRTUALENV,
+    env_manager: str = _EnvManager.VIRTUALENV,
```

Comment:
```suggestion
    env_manager: Literal["virtualenv", "local", "conda"] = _EnvManager.VIRTUALENV,
```

This is more accurate.

Reply:
You can define a type like this if that's helpful:

```
EnvManager = Literal["virtualenv", "local", "conda"]
```